modules/lambda/outputs.tf

```
@@ -1,17 +1,11 @@
 output "lambda_function_arns" {
-  value = tomap({
-    for k, l in aws_lambda_function.lambda : k => l.arn
-  })
+  value = module.lambda_function.*.lambda_function_arn
```

nit: this is the new way to use splat in tf 1.x

```suggestion
  value = module.lambda_function[*].lambda_function_arn
```